src/definitions/elements/icon.less

```diff
@@ -372,4 +374,80 @@ i.icons {
   }
 }
 
+& when ((@variationIconGroups) or (@variationIconCorner))
+   and ((@variationIconBordered) or (@variationIconCircular)) {
+  /*************************************************
+     Bordered/circular with corner or group icons
+  *************************************************/
+  i.bordered.icons,
+  i.circular.icons {
+    width: @borderedSize;
+    height: @borderedSize;
+    box-shadow: @borderedShadow;
+    vertical-align: middle;
+  }
+  i.circular.icons {
+    border-radius: 500em;
+  }
+  i.bordered.icons i.icon:first-child,
+  i.circular.icons i.icon:first-child {
+    position: absolute;
+    transform: translateX(-50%) translateY(-50%);
+  }
+
+  & when (@variationIconInverted) {
+    /* Inverted Icon */
+    i.circular.inverted.icons,
+    i.bordered.inverted.icons {
+      border: none;
+      box-shadow: none;
+    }
```

Comment:
Let's combine those. The selector is the same

Reply:
Oh, lol. Didnt even realize.